packages/web-media-box/src/hls-parser/tags/tagWithAttributesProcessors.ts

```diff
@@ -551,3 +552,18 @@ export class ExtXSessionKey extends EncryptionTagProcessor {
     playlist.sessionKey = this.parseEncryptionTag(tagAttributes) as SessionKey;
   }
 }
+
+export class ExtXContentSteering extends TagWithAttributesProcessor {
+  private static readonly SERVER_URI = 'SERVER-URI';
+  private static readonly PATHWAY_ID = 'PATHWAY-ID';
+
+  protected requiredAttributes = new Set([ExtXContentSteering.SERVER_URI]);
+  protected tag = EXT_X_CONTENT_STEERING;
```

Comment:
_minor_, both should be `readonly`

Reply:
Oh good call, I think I was using the auto fill on the last 2 tags and left that out. Fixing for both.